Userland/Applications/PixelPaint/ImageEditor.cpp

```diff
@@ -203,6 +290,10 @@ void ImageEditor::mousedown_event(GUI::MouseEvent& event)
 
 void ImageEditor::mousemove_event(GUI::MouseEvent& event)
 {
+    m_mouse_position = event.position();
+    update(mouse_indicator_rect_x());
+    update(mouse_indicator_rect_y());
```

Comment:
Maybe wrap those in an `if(m_show_rulers)` so we don't repaint those areas if the rulers are disabled?

Reply:
Sounds like a good idea, will fix tomorrow!